package.json

```
@@ -1,27 +1,53 @@
 {
-  "name": "@safsbe/mental-health-app-monorepo",
+  "name": "mental-health-app",
+  "main": "expo-router/entry",
   "version": "1.0.0",
-  "description": "A proof of concept application created by the SBE Innovation Team, part of the SBE NSF Council.",
-  "main": "index.js",
-  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "android": "expo run:android",
-    "ios": "expo run:ios"
+    "start": "expo start",
+    "reset-project": "node ./scripts/reset-project.js",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "test": "jest --watchAll",
+    "lint": "expo lint"
```

Retain Android App Bundle (AAB) build command.

```suggestion
    "lint": "expo lint",
    "dev-test-android-build": "react-native bundle --dev false --platform android --entry-file app/index.js --bundle-output ./android/app/src/main/assets/index.android.bundle --assets-dest ./android/app/src/main/res/"
```